scripts/deploy-rpc-playbooks.sh

```diff
@@ -30,23 +30,53 @@ cd ${RPCD_DIR}/playbooks/
 
 # configure everything for RPC support access
 if [[ "${DEPLOY_SUPPORT_ROLE}" == "yes" ]]; then
-  run_ansible rpc-support.yml
+    run_ansible rpc-support.yml
 fi
 
 # deploy and configure the ELK stack
 if [[ "${DEPLOY_ELK}" == "yes" ]]; then
-  run_ansible setup-logging.yml
+    run_ansible setup-logging.yml
 fi
 
 # Download the latest release of rpc-maas
 run_ansible maas-get.yml
 
-# deploy and configure RAX MaaS
+# Deploy and configure RAX MaaS
 if [[ "${DEPLOY_MAAS}" == "yes" ]]; then
-  # Run the rpc-maas setup process
-  run_ansible setup-maas.yml
+    # Run the rpc-maas setup process
+    run_ansible setup-maas.yml
 
-  # verify RAX MaaS is running after all necessary
-  # playbooks have been run
-  run_ansible verify-maas.yml
+    # verify RAX MaaS is running after all necessary
+    # playbooks have been run
+    run_ansible verify-maas.yml
+fi
+
+# To send data to the influxdb server, we need to deploy and
+# configure telegraph.
+# By default, telegraph will use log_hosts (rsyslog hosts)
+# to define its influxdb servers.
+# These playbooks need maas-get to have run previously.
+if [[ "${DEPLOY_TELEGRAPH}" == "yes" ]]; then
+    # BUILD_TAG is used as reference for gating, to allow
+    if [[ -z "${BUILD_TAG}" ]]; then
+        # user_zzz_variables are generated at every build, so
+        # we are fine to just echo it.
+        echo 'maas_job_reference: "${BUILD_TAG}"' >> /etc/openstack_deploy/user_zzz_gating_metrics_variables.yml
+        # Telegraph shipping is done to influx nodes belonging to
+        # influx_telegraf_targets | union(influx_all)
+        if [[ -z "${INFLUX_IP}" ]]; then
```

Comment:
To avoid gate mistakes, I created this card: https://rpc-openstack.atlassian.net/browse/LA-354 

in the meantime I think we can merge this.